infra/resources/_modules/app_service_attribute_authority_uat/locals.tf

```diff
@@ -0,0 +1,33 @@
+locals {
+  attribute_authority = {
+    app_settings = {
+      NODE_ENV = "production"
+
+      WEBSITE_NODE_DEFAULT_VERSION        = "18.13.0"
+      WEBSITES_ENABLE_APP_SERVICE_STORAGE = false
+      WEBSITES_PORT                       = 8080
+      SERVER_PORT                         = 8080
+
+      WEBSITE_RUN_FROM_PACKAGE                        = "1"
+      WEBSITE_DNS_SERVER                              = "168.63.129.16"
+      WEBSITE_ADD_SITENAME_BINDINGS_IN_APPHOST_CONFIG = "1"
+
+      # Postgres
+      ATTRIBUTE_AUTHORITY_POSTGRES_DB_SSL_ENABLED = true
+      ATTRIBUTE_AUTHORITY_POSTGRES_DB_URI = var.attribute_authority_postgres_db_admin_connection_string
+
+      # APP INSIGHTS
+      APPINSIGHTS_INSTRUMENTATIONKEY  = var.ai_instrumentation_key
+      APPINSIGHTS_CONNECTION_STRING   = var.ai_connection_string
+      APPINSIGHTS_SAMPLING_PERCENTAGE = var.ai_sampling_percentage
```

Comment:
It is possible to set the following variables in the app service module instead:
- application_insights_connection_string
- application_insights_sampling_percentage (defaults to 5)
```suggestion
```